bridge/client/app/_components/ktb-trigger-sequence/ktb-trigger-sequence.component.html

```diff
@@ -272,7 +272,7 @@ <h2 uitestid="ktb-trigger-headline">Trigger an evaluation for {{ selectedService
           >
             <dt-icon name="calendar"></dt-icon>
           </button>
-          <dt-hint>Has to be before end date</dt-hint>
+          <dt-hint>The start date and time of an evaluation that defaults to now if not filled in.</dt-hint>
```

Comment:
```suggestion
          <dt-hint>The start date and time of an evaluation that defaults to now if not specified.</dt-hint>
```
Or "provided" would work -- filled in is a little slangy and has that whole dangling preposition problem ;-)